### DISCUSSION TITLE
fixed declarations

### DISCUSSION THREAD

**Discussion by tannergooding** [dotnet/csharplang#2186] [Language Ideas]
### Summary

Much in the same way that `using declarations` have been enabled in the language, we should allow `fixed declarations`.

### Motiviation

The motiviations, design, and considerations are essentially the same as for `using declarations` and `fixed` was listed as a possible `future expansion` in the original `using declaration` proposal: https://github.com/dotnet/csharplang/blob/master/proposals/csharp-8.0/using.md#future-expansions

**Comment by tannergooding** [comment]
CC. @jaredpar 

I'm happy to actually fill in the details above, if that is more desirable.

**Comment by Perksey** [comment]
+1

**Comment by john-h-k** [comment]
would also love this much

**Comment by Sergio0694** [comment]
Definitely +1 to this 😄
Would simplify code a lot, especially when you have to do lots of short lived pinnings.

**Comment by nathanAjacobs** [comment]
This would be a great feature, hopefully it is implemented one day.

**Comment by IS4Code** [comment]
Still looking forward to this!
